### PR TITLE
Azure Monitor - Regenerate from updated swagger

### PR DESCRIPTION
**Pull request by srnagar**
This PR uses the swagger in azure-rest-api-specs to regenerate code for logs and metrics clients.

### CODE REVIEW

File: sdk/monitor/azure-monitor-query/swagger/log_query_swagger.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "swagger": "2.0",
   "info": {
-    "version": "v1",
+    "version": "2021-05-19_Preview",
```
Comment (pallavit) [inline sdk/monitor/azure-monitor-query/swagger/log_query_swagger.json:4]: Did the service change their API versioning scheme? ACR is in the similar boat, did you work with them to use the newer scheme here?

**Review by pallavit — APPROVED** [review]
:shipit:

File: sdk/monitor/azure-monitor-query/swagger/log_query_swagger.json (hunk 1 above)
Comment (srnagar) [reply sdk/monitor/azure-monitor-query/swagger/log_query_swagger.json:4]: I authored the swagger for this preview, so, I used the correct format here. I am waiting for feedback from the service team to see if they are okay with the swagger version change.

**Pull request merged**